src/main/java/com/avispl/dal/communicator/polycom/videoos/PolycomVideoOS.java

```diff
@@ -1836,16 +2128,18 @@ private <T> T getJsonProperty(JsonNode json, String property, Class<T> type) {
     @Override
     protected void authenticate() throws Exception {
         // authenticate and get sessionId like "PSN0HppfZap7wtV9MgTeGKLZL+q8q+65Te6g/r61KLqC26+thY"
+        System.out.println("Performing authorization");
         ObjectNode request = JsonNodeFactory.instance.objectNode();
         request.put("user", getLogin());
         request.put("password", getPassword());
-
         JsonNode json = doPost(Constant.URI.SESSION, request, JsonNode.class);
 
         Boolean success = getJsonProperty(json, "success", Boolean.class);
         if (success == null ? false : success) {
             sessionId = getJsonProperty(json.get("session"), "sessionId", String.class);
+            System.out.println("New sessionId " + sessionId);

```

Comment:
I think it's better to avoid using System.out.println for logging, since this involves sensitive info.